Source/WebCore/accessibility/atspi/AccessibilityObjectAtspi.cpp

```diff
@@ -288,8 +288,6 @@ static Atspi::Role atspiRole(AccessibilityRole role)
         return Atspi::Role::Section;
     case AccessibilityRole::Footer:
         return Atspi::Role::Footer;
-    case AccessibilityRole::Form:
-        return Atspi::Role::Form;
```

Comment:
This feels like a pretty significant change for the ATSPI platform. Are we worried about ATs on that platform no longer announcing form, no longer seeing form containers and thus losing important semantics, etc?